cpp/src/io/json/json_column.cu

```diff
@@ -371,7 +371,7 @@ std::vector<std::string> copy_strings_to_host(device_span<SymbolT const> input,
   auto to_host        = [stream](auto const& col) {
     if (col.is_empty()) return std::vector<std::string>{};
     auto const scv     = cudf::strings_column_view(col);
-    auto const h_chars = cudf::detail::make_std_vector_sync<char>(
+    auto const h_chars = cudf::detail::make_std_vector_async<char>(
```

Comment:
This is not necessary, as `h_offsets` will sync. IMO, the cleaner way is to use `async` for both,  then explicitly call `stream.synchronize()`.

Reply:
Right. I made similar decision in other comment too. I will updated it.